### ISSUE TITLE
saveTo returned FilenameMissingException error

### ISSUE THREAD

**Issue by hanifsulong** [stechstudio/laravel-zipstream#17]
Hi, I am using v2.1 and I encountered the same issue as #6 . When I changed to v1.5 with the same code it run without any problem. Are there any solutions to this?

**Comment by hanifsulong** [comment]
Here is the code that I use. Worked with v1.5 but not with higher version

`Zip::create($zipfilename,[
        storage_path('app/temp/shapefile/'.$filename.'.shp'),
        storage_path('app/temp/shapefile/'.$filename.'.shx'),
        storage_path('app/temp/shapefile/'.$filename.'.dbf')
      ])->saveTo(storage_path('app/temp/zip'));`

**Comment by jszobody** [comment]
Are you by chance on Windows? Can you tell me what that `storage_path('app/temp/zip')` evaluates to? I'm particularly interested in the beginning of the path, whether it is a `/` or something like `c:/`.

**Comment by hanifsulong** [comment]
Yeah. I'm using Windows. It is a full path to laragon folder starting with `C:/`.

**Comment by jszobody** [comment]
Ok try this:

```php
use STS\ZipStream\Models\LocalFile;

Zip::create($zipfilename, [
    storage_path('app/temp/shapefile/'.$filename.'.shp'), 
    storage_path('app/temp/shapefile/'.$filename.'.shx'), 
    storage_path('app/temp/shapefile/'.$filename.'.dbf') 
])->saveTo(new LocalFile(storage_path('app/temp/zip')));
```

**Comment by jszobody** [comment]
Or you can upgrade to v2.2 and try again with your original code. I believe that should work for you now, though I don't have a Windows machine myself to try it out. Let me know.

**Comment by hanifsulong** [comment]
I upgraded to v2.2 using the original code, I still got the same error.

The code you gave is not working, I got permission denied error

`use STS\ZipStream\Models\LocalFile;

Zip::create($zipfilename, [
    storage_path('app/temp/shapefile/'.$filename.'.shp'), 
    storage_path('app/temp/shapefile/'.$filename.'.shx'), 
    storage_path('app/temp/shapefile/'.$filename.'.dbf') 
])->saveTo(new LocalFile(storage_path('app/temp/zip')));`

So, I tried to put the filename into the saveTo path and its working perfectly!

`use STS\ZipStream\Models\LocalFile;

Zip::create($zipfilename, [
    storage_path('app/temp/shapefile/'.$filename.'.shp'), 
    storage_path('app/temp/shapefile/'.$filename.'.shx'), 
    storage_path('app/temp/shapefile/'.$filename.'.dbf') 
])->saveTo(new LocalFile(storage_path('app/temp/zip/'.$zipfilename)));`

I'm so glad I can use the latest version, otherwise I have to copy v1.5 from my older project :D
Thanks sir for your respond!
